### PR TITLE
configure plugins in parent

### PR DESCRIPTION
**Pull request by hmdebenque**

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/976/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/976/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/993/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/993/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/994/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/994/)

File: dataprep-backend-service-parent/pom.xml [hunk 1]
```diff
@@ -90,58 +90,81 @@
...
+                                <classifier>lib</classifier>
+                                <classesDirectory>${project.build.outputDirectory}</classesDirectory>
+                                <excludes>
+                                    <exclude>**/application*.properties</exclude>
+                                </excludes>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <goals>
+                                <goal>test-jar</goal>
```
Comment (fhuaulme) [inline dataprep-backend-service-parent/pom.xml]: Does it mean non-service modules (like actions) won't have a test-jar module?

Comment (hmdebenque) [reply dataprep-backend-service-parent/pom.xml]: all modules will have their test jar built. actions module included

Comment (fhuaulme) [reply dataprep-backend-service-parent/pom.xml]: 👍

**Pull request merged**